### ISSUE TITLE
Use the OpenAI Whisper instead of Amazon Transcribe

### ISSUE THREAD

**Issue by khalilgharbaoui** [robertgv/chatgpt-aws#2]
Would it not be beneficial to use OpenAI's Whisper instead of Amazon Transcribe?  

Maybe if that is done right there wont be a need for the S3 Buckets as well  

Wav > OpenAI Whisper > OpenAI ChatGPT > Amazon Polly > Mp3  
  
Instead of:  
  
Wav > S3 Bucket > Amazon Transcribe > S3 Bucket > OpenAI ChatGPT > Amazon Polly > Mp3  
  
Maybe some thoughts or/and ideas on this would be nice?

**Comment by michael7908** [comment]
I think your idea is a great one while installing Whisper on a local machine or Cloud is somewhat challenging... Alas.

**Comment by khalilgharbaoui** [comment]
True, although it can be dockerized since the model is relatively small 🤔 
(i came across this: https://lablab.ai/t/whisper-api-flask-docker)
then deployed as an API endpoint somewhere like https://fly.io
A little bit more hands on... but maybe cheaper, maybe not 😅

Cheers
